### PR TITLE
Added the rest of PM team

### PR DESCRIPTION
**Pull request by elviraux**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/internal-acls/145)

### CODE REVIEW

**Comment by theadactyl** [comment]
Thanks Elvira!

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -397,6 +397,12 @@ orgs:
             - quanjielin   
             - richardsliu
             - vkoukis
+            - elviraux
+            - jbottum
+            - theadactyl
```
Comment (jlewi) [inline github-orgs/kubeflow/org.yaml]: Can you alphabetize please

Comment (elviraux) [reply github-orgs/kubeflow/org.yaml]: ok

**Pull request merged**